docs/source/new-docs/getting-started/installing/basic-env-setup.rst

```diff
@@ -0,0 +1,146 @@
+:orphan:
+
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
+
+Installing
```

Comment:
If this is going to be our "official" installatin method that we are sending out for people to use this week, then we will need to update the README.md to point here for installation.

Do we want to update this now?

Reply:
This is what I tested, but I'm fine sending out the old instructions and waiting on this if we still have things to discuss